runtime/doc/luaref.txt

```diff
@@ -20,72 +20,7 @@
-           2.5.2   Relational Operators............|luaref-langRelOp|
-           2.5.3   Logical Operators...............|luaref-langLogOp|
-           2.5.4   Concatenation...................|luaref-langConcat|
-           2.5.5   The Length Operator.............|luaref-langLength|
-           2.5.6   Precedence......................|luaref-langPrec|
-           2.5.7   Table Constructors..............|luaref-langTableConst|
-           2.5.8   Function Calls..................|luaref-langFuncCalls|
-           2.5.9   Function Definitions............|luaref-langFuncDefs|
-           2.6   Visibility Rules..................|luaref-langVisibRules|
-           2.7   Error Handling....................|luaref-langError|
-           2.8   Metatables........................|luaref-langMetatables|
-           2.9   Environments......................|luaref-langEnvironments|
-           2.10   Garbage Collection...............|luaref-langGC|
-           2.10.1   Garbage-Collection Metamethods.|luaref-langGCMeta|
-           2.10.2   Weak Tables....................|luaref-langWeaktables|
-           2.11   Coroutines.......................|luaref-langCoro|
-
-           3   THE APPLICATION PROGRAM INTERFACE...|luaref-api|
-           3.1   The Stack.........................|luaref-apiStack|
-           3.2   Stack Size........................|luaref-apiStackSize|
-           3.3   Pseudo-Indices....................|luaref-apiPseudoIndices|
-           3.4   C Closures........................|luaref-apiCClosures|
-           3.5   Registry..........................|luaref-apiRegistry|
-           3.6   Error Handling in C...............|luaref-apiError|
-           3.7   Functions and Types...............|luaref-apiFunctions|
-           3.8   The Debug Interface...............|luaref-apiDebug|
-
-           4   THE AUXILIARY LIBRARY...............|luaref-aux|
-           4.1   Functions and Types...............|luaref-auxFunctions|
-
-           5   STANDARD LIBRARIES..................|luaref-lib|
-           5.1   Basic Functions...................|luaref-libBasic|
-           5.2   Coroutine Manipulation............|luaref-libCoro|
-           5.3   Modules...........................|luaref-libModule|
-           5.4   String Manipulation...............|luaref-libString|
-           5.4.1   Patterns........................|luaref-libStringPat|
-           5.5   Table Manipulation................|luaref-libTable|
-           5.6   Mathematical Functions............|luaref-libMath|
-           5.7   Input and Output Facilities.......|luaref-libIO|
-           5.8   Operating System Facilities.......|luaref-libOS|
-           5.9   The Debug Library.................|luaref-libDebug|
-
-           A   BIBLIOGRAPHY........................|luaref-bibliography|
-           B   COPYRIGHT & LICENSES................|luaref-copyright|
-           C   LUAREF DOC..........................|luaref-doc|
-
+Type |gO| to see the table of contents.
```

Comment:
Just use `gO`, don't hardcode ToC. (Future: could consider dynamically adding ToC to the document.